Valet/SynchronizableAccessibility.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SynchronizableAccessibility.swift
+//  Valet
+//
+//  Created by Dan Federman on 9/17/17.
+//  Copyright © 2017 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+
+public enum SynchronizableAccessibility: CustomStringConvertible {
```

Comment:
possible to protocolize instead?

Reply:
also, possibly better spot (maybe not): internal enums to the two classes where we can only expose a subset like this? (synchronizable, secureenclave)